mmv1/third_party/terraform/services/secretmanager/resource_secret_manager_secret_test.go.tmpl

```diff
@@ -464,6 +464,60 @@ func TestAccSecretManagerSecret_updateBetweenTtlAndExpireTime(t *testing.T) {
 	})
 }
 
+func TestAccSecretManagerSecret_DeletionProtection(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name             string
+		deleteProtection string
+	}{
+		{
+			name:             "True",
+			deleteProtection: "true",
+		},
+		{
+			name:             "False",
+			deleteProtection: "false",
+		},
+	}
+
+	for _, tc := range testCases {
```

Comment:
no, these are not separate test cases. This needs to be one test case, \which first sets deletion protection to true and then updates it to false. Otherwise, the resource created in the first test can't ever be deleted.